cfgov/cfgov/urls.py

```diff
@@ -469,7 +470,9 @@
 
 if 'selfregistration' in settings.INSTALLED_APPS:
     from selfregistration.views import CompanySignup
-    pattern = url(r'^company-signup/', CompanySignup.as_view())
+    pattern = flagged_url('WAGTAIL_COMPANY_SIGNUP', r'^company-signup/',
+                          CompanySignup.as_view(), state=False,
+                          fallback=lambda req: ServeView.as_view()(req, req.path)) # noqa
```

Comment:
This could've been:

```python
    pattern = flagged_url(
        'WAGTAIL_COMPANY_SIGNUP', 
        r'^company-signup/',
        CompanySignup.as_view(), 
        state=False,
        fallback=lambda req: ServeView.as_view()(req, req.path)
    )
```

Without the `# noqa` 😜

Reply:
Confession: I don't really understand the continuation-indentation rules, I thought everything had be lined up with the open parens on flagged_url(

Reply:
PEP 8 will accept either:
- everything indented to the parens (with the first parameter following the parens) like you have
- everything indented one level after an open bracket (with the first parameter on the next line from the parens) like Will shows

There are some cases where `flake8` will flag something as "insufficient indentation" but that's generally in cases involving strings, not parameters. Hope this helps!

Reply:
Thanks, @hillaryj!